AlphaWallet/KeyManagement/EtherKeystore.swift

```diff
@@ -11,10 +11,13 @@ enum EtherKeystoreError: LocalizedError {
     case protectionDisabled
 }
 
-///We use ECDSA keys, achieving symmetric encryption based on Diffie-Hellman to encrypt the HD wallet seed and raw private keys and store the ciphertext in the keychain
+///We use ECDSA keys (created and stored in the Secure Enclave), achieving symmetric encryption based on Diffie-Hellman to encrypt the HD wallet seed and raw private keys and store the ciphertext in the keychain.
 ///
-///The ECDSA key is stored in the secure enclave, not requiring user presence
-///The ciphertext is stored in the keychain, not requiring user presence at first, but once user has backed up, requires user presence
+///There are 2 sets of (ECDSA key and ciphertext) for each Ethereum raw private key or HD wallet seed. 1 set is stored requiring user presence for access and the other doesn't. The second set is needed to ensure the user has does not lose access to the Ethereum raw private key (or HD wallet seed) when they delete their iOS passcode. Once the user has verified that they have backed up their wallet, they can choose to elevate the security of their wallet which deletes the set of (ECDSA key and ciphertext) that do not require user-presence.
+///
+///Technically, having 2 sets of (ECDSA key and ciphertext) for each Ethereum raw private key or HD wallet seed may not be required for iOS. But it is done:
+///(A) to be confident that we cause the user to lose access to their wallets and
```

Comment:
Don't you mean "to be confident that we _don't_ cause the user to lose access"? ☺️